docs/modules/ROOT/pages/guides/ogm-subscriptions.adoc

```diff
@@ -0,0 +1,241 @@
+[[ogm-subscriptions]]
+= How to use the OGM with subscriptions
+
+The Neo4j GraphQL Library can be used in conjunction with the Neo4j OGM in order to extend the Library's functionalities, or to take advantage of the xref:ogm/private.adoc[`@private` directive]. 
+
+== Usage
+
+This section shows you how to use subscriptions with the OGM inside custom resolvers.
+
+=== Prerequisites
+
+Considering the following type definitions:
+[source, javascript, indent=0]
+----
+const typeDefs = `#graphql
+    type User {
+      email: String!
+      password: String! @private
+    }
+`;
+----
+
+A subscriptions plugin instance to be shared between the Library and the OGM constructors:
+[source, javascript, indent=0]
+----
+const subscriptionsPlugin = new Neo4jGraphQLSubscriptionsSingleInstancePlugin();
+----
+
+And a server that supports subscriptions, as per the xref:subscriptions/getting-started.adoc#setting-up-server[subscriptions reference usage].
```

Comment:
The reference is not pointing to another section in this guide, but on a different page in the documentation (the page dedicated to Subscriptions).

Reply:
Ah yes by "this guide" is the GraphQL docs. But now that you said it might look ambiguous! Maybe something like this

```suggestion
. Set up a server that supports subscriptions according to the xref:subscriptions/getting-started.adoc#setting-up-server[subscriptions reference usage].
```